x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/columns.tsx

```diff
@@ -148,8 +147,36 @@ export const getCasesColumns = (
           return getEmptyTagValue();
         },
       },
+  {
+    name: 'ServiceNow Incident',
+    render: (theCase: Case) => {
+      if (theCase.id != null) {
+        return <ServiceNowColumn theCase={theCase} />;
+      }
+      return getEmptyTagValue();
+    },
+  },
   {
     name: 'Actions',
     actions,
   },
 ];
+
+interface Props {
+  theCase: Case;
+}
+
+const ServiceNowColumn: React.FC<Props> = ({ theCase }) => {
+  const { hasDataToPush } = useGetCaseUserActions(theCase.id);
```

Comment:
Ok, that works for me but we should create an API or adding this logic in _find cases. Let's add an issue to track it.

Reply:
what do you mean track the loading??

Reply:
showing that the data is loading

Reply:
`useGetCaseUserActions` is a rest call